nasa/cFE
# Fix #557, OSAL Guide Scrub

**Describe the contribution**
Fix #557, see issue for list of changes.
These are the changes required in the cFE repo to scrub the OSAL guide

Note https://github.com/nasa/cFE/commit/8845570f5044a984e07b848f75090d27229c22f6 is the commit related to this issue.

**Testing performed**
Steps taken to test the contribution:
1. Nominal build (native) and run, no issues
1. Make usersguide and confirmed no warnings in build/doc/warnings
1. cd build/doc/users_guide/latex
1. make
1. confirm completes and guide looks ok
1. Make osalguide and confirmed no warnings in build/doc/warnings
1. cd build/doc/osalguide/latex
1. make
1. confirm completes and guide looks ok

**Expected behavior changes**
No behavior change other than warnings resolved for doc build

**System(s) tested on**
 - Hardware: cFS Dev Server 3
 - OS: Ubuntu 18.04
 - Versions: Bundle w/ the commits below

**Additional context**
None

**Third party code**
None

**Contributor Info - All information REQUIRED for consideration of pull request**
Jacob Hageman - NASA/GSFC
